app/models/reservation.rb

```diff
@@ -321,6 +317,23 @@ def update(current_user, new_params, new_notes) # rubocop:disable all
     end
   end
 
+  def add_notes(current_user, contents)
+    return false if contents.empty?
```

Comment:
Note that while you specify the error if the notes are too long, you don't specify the error if the notes are too *short*. If we're just going to stick the logic here, I think it would be better if we just checked for `contents.length <= max_length && contents.length >= min_length` and just had a general error message for incorrectly sized notes. Alternatively, if you want to use a separate model, we should implement the error message(s) in the `Contents` model and then simply pass them through here (and then we could conceivably distinguish between different notes errors).